packages/v4/patternfly-docs/content/design-guidelines/components/tree-view/tree-view.md

```diff
@@ -92,7 +92,8 @@ A tree view in a [primary-detail](/demos/primary-detail) or [drawer](/components
 
 <img src="./img/treeview-primarydetail-drawer.png" alt="Tree view in primary detail drawer usage" width="1500" />
 
-## Compact tree view
+## Variation
```

Comment:
And then if you want to organize this how you did the tables/data lists, then you could rename this section "Spacing" instead of "Variations"